lang/src/org/partiql/lang/eval/visitors/GroupKeyReferencesVisitorTransform.kt

```diff
@@ -47,6 +47,10 @@ import org.partiql.lang.planner.transforms.errAstNotNormalized
  *
  * ```
  */
+@Deprecated(
+    "This class is subject to removal.",
+    level = DeprecationLevel.WARNING
+)
 public class GroupKeyReferencesVisitorTransform(
```

Comment:
```suggestion
class GroupKeyReferencesVisitorTransform(
```

Reply:
We're overall trying to get to Explicit API mode -- which would include adding visibility modifiers.